includes/functions.php

```diff
@@ -34,12 +34,46 @@
  *
  * @param array $args Arguments for fetching stories.
  *
- * @return string|void
+ * @return void
  */
 function stories( $args = [] ) {
-	$story_query = new Story_Query( $args );
-	//phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
-	echo $story_query->render();
+	$attrs = [];
+
+	if ( empty( $args ) ) {
+		//phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
+		echo get_plugin_instance()->customizer->render_stories();
+	} else {
+		$default_attributes = [
+			'view_type',
+			'number-of-stories',
+			'number_of_columns',
+			'show_title',
+			'show_excerpt',
+			'show_author',
+			'show_date',
+			'show_stories_archive_link',
+			'stories_archive_label',
+			'list_view_image_alignment',
+			'class',
+			'circle_size',
+		];
+
+		$intersection = array_intersect_key( $args, array_flip( $default_attributes ) );
+
+		/**
+		 * Create a new array of attributes.
+		 */
+		if ( $intersection ) {
+			foreach ( $intersection as $key => $val ) {
+				$attrs[ $key ] = $val;
+				unset( $args[ $key ] );
+			}
+		}
```

Comment:
Could `wp_parse_arrgs()` be used here?

Reply:
We are doing this because we need to pass `attributes` and `query_args` to `Story_Query` object.

However; we are passing single array of arguments to function. This helps us separating `attributes` from `query args`, so that we will be able to pass both `attrs` and `query_args` separately.

Reply:
Would be good to document this from the start, even if it's just as a PR comment :-)

I have a hard time figuring out in which case `$attrs` will _not_ be empty. Can you provide some examples?

Also, this causes PHP warnings:

```
Warning: array_flip(): Can only flip string and integer values, entry skipped
```

> However; we are passing single array of arguments to function.

Why do we need a single array only? Can't we just pass them separately? Seems way easier to maintain.

Then we also don't need a `get_stories_object()` helper function because we can just do `new Story_Query( $attrs, $args )`